lib/module/xkcd.go

```diff
@@ -0,0 +1,88 @@
+package module
+
+import (
+	"fmt"
+	"github.com/davidscholberg/irkbot/lib/configure"
+	"github.com/davidscholberg/irkbot/lib/message"
+	"github.com/nishanths/go-xkcd"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const apiUrlFmtDefine = "https://relevantxkcd.appspot.com/process?%s"
+
+func Helpxkcd() []string {
+	s := "xkcd <search> - find a xkcd comic relevant to the search term"
+	return []string{s}
+}
+
+//Perform the actual GET and return the resulting body as a string
+func get(apiURL string) (string, error) {
+	response, err := http.Get(apiURL)
+	if err != nil {
+		return "", err
+	}
+	defer response.Body.Close()
+	bodyBytes, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return "", err
+	}
+	bodyString := string(bodyBytes)
+	return bodyString, nil
+}
+
+//Parse the body string for the comic number we want
+func parseString(bodyString string) string {
+	bodyStrings := strings.Split(bodyString, "\n")
+	if len(bodyStrings) < 3 {
+		return "error"
+	}
+	spacedStrings := strings.Fields(bodyStrings[2])
+	return spacedStrings[0]
+}
+
+//Method called on xkcd command, named funky so as not to collide with xkcd-go
+func getXKCD(cfg *configure.Config, in *message.InboundMsg, actions *Actions) {
+	comicMsg := "enter a search term, dipthong"
```

Comment:
I spelled it wrong... https://en.wikipedia.org/wiki/Diphthong

My Latin teacher in HS used to say it would make a good insult, but I can change it to something more generic.